src/parser/Abstract_Class_Contextual_Parser.hh

```diff
@@ -0,0 +1,74 @@
+//----------------------------------*-C++-*----------------------------------//
+/*!
+ * \file   parser/Abstract_Class_Contextual_Parser.hh
+ * \author Kent Budge
+ * \brief  Define class Abstract_Class_Contextual_Parser
+ * \note   Copyright (C) 2016 Los Alamos National Security, LLC.
+ *         All rights reserved.
+ */
+//---------------------------------------------------------------------------//
+// $Id$
```

Comment:
Please remove the $Id$ lines and the extra separator lines in your new files.  These are not really useful when working with git.  If you have a template that you use to generate these files, you may want to update those files as well.